src/main/proto/com/kodypay/grpc/pay/v1/pay.proto

```diff
@@ -123,9 +123,9 @@ message PaymentDetailsRequest {
 // requires X-API-Key header with 'API Key' value
 message RefundRequest {
   string store_id = 1; // UUID of store
-  string order_id = 2;
+  string payment_id = 2; // UUID idempotency key. Will be generated by client.
   string amount = 3; // amount in BigDecimal/2.dp (0.00)
-  optional string idempotency_key = 4; // UUID idempotency key. Will be generated by client.
```

Comment:
We should keep `idempotency_key` in the refund request - but rename `order_id` to `payment_id`, we _can_ rename it to be `idempotency_uuid` to keep it consistent with `PayRequest`